libxcursor.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: libxcursor
+  version: 1.2.1
+  epoch: 0
+  description: X cursor management library
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - xorgproto
+      - util-macros
+      - libx11-dev
+      - libxrender-dev
+      - libxfixes-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 46c143731610bafd2070159a844571b287ac26192537d047a39df06155492104
+      uri: https://www.x.org/releases/individual/lib/libXcursor-${{package.version}}.tar.xz
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
```

Comment:
(not blocking) Is the spacing intentional? Is `yam` doing that? 🤔

Reply:
Yes